src/Core/Components/DataGrid/FluentDataGrid.razor.cs

```diff
@@ -212,8 +213,21 @@ public partial class FluentDataGrid<TGridItem> : FluentComponentBase, IHandleEve
     /// Gets or sets the content to render when <see cref="Loading"/> is true.
     /// A default fragment is used if loading content is not specified.
     /// </summary>
-    [Parameter]
-    public RenderFragment? LoadingContent { get; set; }
+    [Parameter] public RenderFragment? LoadingContent { get; set; }
+
+    /// <summary>
+    /// Uses JS to auto-fit the columns to the grid width.
```

Comment:
Can you give a more explicit (and less technical: no `JS`) documentation to explain what `auto-fit` means?

Reply:
I updated the summary in the code. Not sure if that's what you are looking for?